data/json/items/armor/ammo_pouch.json

```diff
@@ -32,7 +33,8 @@
     "description": "A fabric ammo pouch that can be strapped to your chest which is capable of holding a single magazine close at hand.",
     "weight": "140 g",
     "volume": "250 ml",
-    "price": 2500,
+    "price": 1250,
+    "price_postapoc": 2500, 
```

Comment:
```suggestion
    "price_postapoc": 2500,
```

Reply:
I'm still learning so does the highlight mean you added a space or deleted a space?